src/declarativetabmodel.cpp

```diff
@@ -378,6 +378,22 @@ void DeclarativeTabModel::updateActiveTab(const Tab &activeTab, bool loadActiveT
     }
 }
 
+void DeclarativeTabModel::setWebContainer(DeclarativeWebContainer *webContainer)
```

Comment:
Just a note: I think there is no need to register tabModels as QML types nowadays thus this method can be dropped with QML type registration altogether.